pkg/deployment/deployment.go

```diff
@@ -265,24 +268,36 @@ func ConditionalDeploy(
 				readyErrorMessage,
 				err.Error()))
 			return err
-		} else if ansibleEEJob.Status.Succeeded > 0 {
+		}
+
+		if ansibleEE.Status.JobStatus == ansibleeev1alpha1.JobStatusSucceeded {
 			log.Info(fmt.Sprintf("Condition %s ready", readyCondition))
 			status.Conditions.Set(condition.TrueCondition(
 				readyCondition,
 				readyMessage))
-		} else if ansibleEEJob.Status.Failed > 0 {
+			return nil
+		}
+
+		if ansibleEE.Status.JobStatus == ansibleeev1alpha1.JobStatusRunning || ansibleEE.Status.JobStatus == ansibleeev1alpha1.JobStatusPending {
+			log.Info(fmt.Sprintf("AnsibleEE job is still running: Execution: %s, Status: %s", ansibleEE.Name, ansibleEE.Status.JobStatus))
```

Comment:
This might be a nit pick but JobStatusPending does not mean the job is running (registered by not yet executed) probably we should rephrase the error to `job is not yet completed` or should use a different message (`running` vs `pending`)

Reply:
Yeah, that's a good point. Would be less confusing if it said job is not yet completed.